Gemfile.lock

```diff
@@ -121,36 +122,35 @@ GEM
       activesupport (>= 3.0.0)
       uniform_notifier (~> 1.11)
     byebug (11.1.3)
-    capistrano (3.14.0)
+    capistrano (3.14.1)
       airbrussh (>= 1.0.0)
       i18n
       rake (>= 10.0.0)
       sshkit (>= 1.9.0)
-    capistrano-bundler (1.6.0)
+    capistrano-bundler (2.0.1)
       capistrano (~> 3.1)
-    capistrano-rails (1.5.0)
+    capistrano-rails (1.6.1)
       capistrano (~> 3.1)
-      capistrano-bundler (~> 1.1)
+      capistrano-bundler (>= 1.1, < 3)
     capistrano-rails-console (2.3.0)
       capistrano (>= 3.5.0, < 4.0.0)
       sshkit-interactive (~> 0.3.0)
     case_transform (0.2)
       activesupport
-    code_analyzer (0.5.1)
+    code_analyzer (0.5.2)
       sexp_processor
     codeclimate-engine-rb (0.4.1)
       virtus (~> 1.0)
     coderay (1.1.3)
     coercible (1.0.0)
       descendants_tracker (~> 0.0.1)
-    concurrent-ruby (1.1.6)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.1.7)
```

Comment:
### [concurrent-ruby](http://www.concurrent-ruby.com)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.1.6 → 1.1.7

[_[change-log](https://github.com/ruby-concurrency/concurrent-ruby/blob/master/CHANGELOG.md), [source-code](https://github.com/ruby-concurrency/concurrent-ruby)_]





<details>
<summary>Commits</summary>

A change of **42** commits. See the full changes on [the compare page](https://github.com/ruby-concurrency/concurrent-ruby/compare/v1.1.6...v1.1.7).

These are the first 10 commits:
- (5652d30) [Remove unused line](https://github.com/ruby-concurrency/concurrent-ruby/commit/5652d30bf584268cc7a37c085f108796bc71e154)
- (50917ff) [Merge pull request #852 from fzakaria/remove-unused-line](https://github.com/ruby-concurrency/concurrent-ruby/commit/50917ff64fcc6e8b40bc53c870392d70abada705)
- (466f065) [Fix argument prefix warning](https://github.com/ruby-concurrency/concurrent-ruby/commit/466f0653f9d842f1c93319dc95a1565330e329c5)
- (2b5fd71) [Merge pull request #854 from bjfish/fix-argument-prefix-warn…](https://github.com/ruby-concurrency/concurrent-ruby/commit/2b5fd71683789da58955156bb496722de67fd899)
- (3bdbefb) [CI: Use JRuby 9.2.10.0](https://github.com/ruby-concurrency/concurrent-ruby/commit/3bdbefb3e7be3afc2164923f50b5c02607b6c498)
- (478c64e) [Do not allow failures on JRuby 9.2.10.0 Latest on Java 11](https://github.com/ruby-concurrency/concurrent-ruby/commit/478c64ee1825e8e405ae9df1aa0db52c5c90081d)
- (85f72c8) [Merge pull request #855 from olleolleolle/patch-2](https://github.com/ruby-concurrency/concurrent-ruby/commit/85f72c81583e9d551c43a604526ec28d747ff0b6)
- (c92d11e) [Introduce ThreadPoolExecutor without a Queue](https://github.com/ruby-concurrency/concurrent-ruby/commit/c92d11e6974161520a0cbeafdc60bf6d15191961)
- (658ef56) [Simply make queue always false when @synchronous](https://github.com/ruby-concurrency/concurrent-ruby/commit/658ef56620e61b907d273e5233be5149591aa93c)
- (bf824c9) [remove whitespace](https://github.com/ruby-concurrency/concurrent-ruby/commit/bf824c9c9121783a157465216faafd7a2e20938a)


</details>